site/static/compare.html

```diff
@@ -1040,7 +1042,14 @@ <h2>Comparing <span id="stat-header">{{stat}}</span> between <span id="before">{
             window.location.search = params.toString();
         }
 
-        loadState(makeData);
+        app.mount('#app');
+
+        document.getElementById("filters-toggle").onclick = (e) => {
```

Comment:
Maybe this should be in `mounted` rather than just after calling `mount` (I'm not sure if `mount` is synchronous?). But it seems to work.

Reply:
These elements are not controlled by vue so there's not really a need to do this in `mounted`.